OpenKeychain/src/main/java/org/sufficientlysecure/keychain/pgp/CanonicalizedSecretKey.java

```diff
@@ -319,6 +320,28 @@ public RSAPrivateCrtKey getCrtSecretKey() throws PgpGeneralException {
         return (RSAPrivateCrtKey)retVal;
     }
 
+    // For use only in card export; returns the secret key.
```

Comment:
I took some time to get back into the code and you are right. For now keep the methods where they are but prefix them with ``securityToken``, such as ``getSecurityTokenECSecretKey``. Please also do this in ``CanonicalizedPublicKey``.

Also rename ``getCrtSecretKey`` to ``getSecurityTokenRSASecretKey``.